src/fut/mod.rs

```diff
@@ -44,7 +44,38 @@ use actor::Actor;
 /// Trait for types which are a placeholder of a value that may become
 /// available at some later point in time.
 ///
-/// This is similar to `futures::Future` trait, except it works with `Actor`
+/// `ActorFuture` is very similar to a regular `Future`, except for the fact that `map`ing over it
```

Comment:
> `map`ing over 

I think it is better to re-phrase it like

>combinators accepts closures with third arguments: reference to actor, context and result

Or maybe something along these lines.
I think we should prefer to be generic here rather than using specific combinator `map`